### PR TITLE
yet more static theme wizard changes

### LINKED ISSUE #7398: Theme generator changes, part deux
The new color selectors are really great. Love playing with the transparency.

**Wizard**
- [x] The "Select a header image" title has no spacing.
header-margin

- [x] The text fields for the color selectors are wider than necessary. Reduce their length to ideally align with the browser preview below.
- [x] Add a margin to top the tooltip buttons in the color selectors. The spacing at the top and bottom of the buttons should be similar.
help-button-margin

- [x] "Header area text" -> "Header area text and icons"
- [x] "Header area text" help text -> "The color of the text and icons in the header area, except the active tab."
- [x] "Toolbar area text" -> "Toolbar area text and icons"
- [x] "Toolbar area text" help text -> "The color of the text and icons in the toolbar and the active tab."
- [ ] The only required field should be the theme name. However, the button shouldn't be enabled until a header image is uploaded or a color field has a non-default value.
- [x] The header image preview (after uploading) needs some size restrictions. A big square image takes more of the page and pushes everything down.

**Description step**

- [ ] The summary field covers the whole width of the page, which makes it partially disappear to the right if the window isn't very wide.  Not sure if this is consistent with other upload flow fields, but ideally it shouldn't be so wide. 
- [x] There's no indication that the license fields are required.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #7398 

I've found a quite serious flaw that got introduced at some point - the header image preview is only accurate with a persona-sized image (200px high, and at least 680px wide, anyway).  These fixes arem't connected (but get it the way) so I'm going to fix that in a follow-up.

### CODE REVIEW

File: static/css/devhub/static-theme.less [hunk 1]
```diff
@@ -16,7 +16,7 @@
         }
     }
     h3 {
-        font-size: 1.1em;
+        font-size: 12pt;
```
Comment (diox) [inline static/css/devhub/static-theme.less]: `pt` is an odd choice of unit here, since we're not printing. Why the change ? I don't see anything regarding this in the issue.

Comment (eviljeff) [reply static/css/devhub/static-theme.less]: I think I meant px.  I changed it because I made `Select a header image for your theme` a `<h3>` also to get equal spacing; but as it's inside a `<label>` that has `1.1em` css rule, and em is a relative size, the `<h3>` inside the label ended up bigger than the others.

Comment (eviljeff) [reply static/css/devhub/static-theme.less]: i.e. stupid css.

Comment (diox) [reply static/css/devhub/static-theme.less]: Ok, that makes sense, but definitely change it to `px` then :)

Comment (eviljeff) [reply static/css/devhub/static-theme.less]: https://github.com/mozilla/addons-server/pull/7417/commits/552784611cbae82b3b8e090a865803d24bd757cd
